src/org/camunda/latera/bss/connectors/Imprint.groovy

```diff
@@ -36,7 +36,7 @@ class Imprint {
     )
   }
 
-  def print(CharSequence template, Map data = [:]) {
+  def print(CharSequence template, Map data = [:], Boolean convertToPDF = false) {
     logger.info('Printing begin...')
```

Comment:
i wonder how it looks when you have thousands of requests.. :)

Reply:
What do you mean?